paper.qmd

```diff
@@ -50,6 +47,20 @@ Reconciling topologically different linestrings is a fundamental challenge in sp
 
 > "Due to the complexity and limitations of existing methods, planners and analysts often have to employ a heavily manual conflation process, which is time‐consuming and often prohibitively expensive." Lei, Ting; Lei, Zhen (2019).
 
+## Definitions
+
+The purpose of this section is to provide clear and concise definitions of terminologies used in the description of this algorithm. 
+
+We will use the simple feature access standards definitions of geometric primitives for the sake of consistency. In our algorithm we make extensive use of **Line** and **LineString** geometric primitives. A Line is defined by two **Point**s that are (x, y) coordinate pairs. A LineString is composed of 2 or more Points and "each consecutive pair of Points defines a Line segment." (@sfa FIXME cite). Each LineString is referred to as a **feature**. An array of features is referred to as a **FeatureCollection** (@geojson_spec). 
+
+The objective of this algorithm is to identify matches between two LineString FeatureCollections and measure the length of the match. A **match** is a correspondence between features of separate FeatureCollections (@lei_lei_19).
+
+The terminology to refer to the two FeatureCollections is inconsistent. For example, Zhang (@zhang_2009) refers to the two FeatureCollections as the **reference** and **target** whereas the Java Conflation Suite (JCS) (@jcs) refers to these as the **reference** and **subject**. 
```

Comment:
Inconsistent *in the literature*.

Worth saying that so people don't think we're deliberately being inconsistent in the paper (which we won't be nothing but clear use of terminology in this one)!